CONFIGURATION.md

```diff
@@ -0,0 +1,182 @@
+| `wave.build.logs.prefix` | Prefix to be used for build process log files in S3 bucket.                                       |        | true     |
+| `wave.build.logs.maxLength` | Maximum number of bytes that can be read from a log file. Log will be truncated after this limit. | `100000`        | false    |
+
+
+### Kubernetes configuration for container build process
+
+**Note**: only applies when using Kubernetes.
+
+| Variable                                   | Description                                              | Default Value / Environment Variable | Optional |
+|--------------------------------------------|----------------------------------------------------------|---------------------------------------|----------|
+| `wave.build.k8s.namespace`                 | Kubernetes namespace where to run wave build pods. | | false    |
+| `wave.build.k8s.storage.claimName`         | Volume claim name for wave build kubernetes pods.              |     | true     |
+| `wave.build.k8s.storage.mountPath`         | Volume mount path on wave build Kubernetes pods.              |        | true     |
+| `wave.build.k8s.labels`                    | Labels to set on wave build kubernetes pods.                              |  | true     |
+| `wave.build.k8s.node-selector`             | Node selector configuration for wave build kubernetes pods.                   |  | true     |
+| `wave.build.k8s.service-account`           | Kubernetes service account name to be used by wave.        |  | true     |
+| `wave.build.k8s.resources.requests.cpu`    | Amount of [CPU resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes) to allocate to wave build processes, e.g. `2` or `1500m`. |                        | true     |
+| `wave.build.k8s.resources.requests.memory` | Amount of [memory resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes) to allocate to wave build processes, e.g. `3Gi` or `2000Mi`.           |                        | true     |
+
+
+## Container scan process configuration
+
+| Variable                      | Description                                                                                        | Default Value  / Environment Variable | Optional |
+|-------------------------------|----------------------------------------------------------------------------------------------------|---------------------------------------|----------|
+| `wave.scan.enabled`                   | Enable or disable vulnerability scanning.                                                          | `false`                               | false    |
+| `wave.scan.severity`                  | [Severity level](https://aquasecurity.github.io/trivy/v0.22.0/vulnerability/examples/filter/) for vulnerability scanning. e.g. `MEDIUM,HIGH,CRITICAL`.                        | `LOW,MEDIUM,HIGH,CRITICAL`            | true     |
```

Comment:
```suggestion
| `wave.scan.severity`                  | [Severity level](https://aquasecurity.github.io/trivy/v0.22.0/vulnerability/examples/filter/) for vulnerability scanning, e.g. `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`.                        | `LOW`            | true     |
```

I guess the default is not ALL values, right?

Reply:
Ok, makes sense then. Rephrasing the description may help.

Reply:
done